composer.json

```
@@ -90,5 +89,6 @@
         "branch-alias": {
             "dev-master": "4.x-dev"
         }
-    }
+    },
+    "minimum-stability": "beta"
```

Remove `minimum-stability` flag on Symfony 7 release
```suggestion
    }
```